prac_07/programming_language.py

```diff
@@ -0,0 +1,48 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Programming Language class with tests.
+"""
+
+
+class ProgrammingLanguage:
+    """Represent information about a programming language."""
+
+    def __init__(self, name, typing, reflection, year, pointer):
```

Comment:
I suggest the name pointer_arithmetic instead of pointer. Not only is it from the problem domain, but also less confusing if you are coming from other programming languages which have pointers.